Source/WebCore/page/scrolling/ScrollingCoordinatorTypes.h

```diff
@@ -140,8 +140,13 @@ enum class KeyboardScrollAction : uint8_t {
 };
 
 struct RequestedKeyboardScrollData {
-    KeyboardScrollAction action;
-    std::optional<KeyboardScroll> keyboardScroll;
+    KeyboardScrollAction action { KeyboardScrollAction::StartAnimation };
+    std::optional<KeyboardScroll> keyboardScroll { };
```

Comment:
No need to explicitly initialize a `std::optional<>`. It has its own constructor.